cli/exchange/nmp.go

```diff
@@ -118,6 +119,27 @@ func NMPAdd(org, credToUse, nmpName, jsonFilePath string, appliesTo, noConstrain
 		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("The node management policy has no constraints which might result in the management policy being deployed to all nodes. Please specify --no-constraints to confirm that this is acceptable."))
 	}
 
+	// Validate the AgentAutoUpgrade job if it was defined
+	if nmpFile.AgentAutoUpgradePolicy != nil {
+
+		// Ensure that a manifest was specified
+		fullManifest := nmpFile.AgentAutoUpgradePolicy.Manifest
+		if fullManifest == ""{
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("An AgentAutoUpgradePolicy was defined, but a manifest was not defined. Please specify a manifest that is stored in the CSS before attempting to add an NMP with an AgentAutoUpgradePolicy."))
+		}
+
+		// Ensure the specified manifest exists in the CSS
+		manOrg, manifest := cliutils.TrimOrg(org, fullManifest)
+		urlPath := "api/v1/objects/" + manOrg + "?filters=true"
+		filterURLPath := fmt.Sprintf("&objectType=%s&objectID=%s", exchangecommon.AU_MANIFEST_TYPE, manifest)
+		fullPath := urlPath + filterURLPath
+		var manifestsMeta []common.MetaData
+		httpCode := cliutils.ExchangeGet("Model Management Service", cliutils.GetMMSUrl(), fullPath, cliutils.OrgAndCreds(org, credToUse), []int{200, 404}, &manifestsMeta)
+		if httpCode == 404 {
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("An AgentAutoUpgradePolicy was defined with manifest '%s' which does not exist in org %s.", fullManifest, manOrg))
```

Comment:
Please add the org in the message. If I put IBM/manifest1 in the NMP json file, the error message should say manifest1 from IBM org orIBM/manifest1 .